spynnaker/pyNN/models/neural_projections/connectors/abstract_connector.py

```diff
@@ -108,37 +108,27 @@ def _check_parameter(self, values, name, allow_lists):
         :param bool allow_lists:
         """
         if (not numpy.isscalar(values) and
-                not (isinstance(values, RandomDistribution)) and
-                not hasattr(values, "__getitem__")):
+                not (isinstance(values, RandomDistribution))):
             raise SpynnakerException("Parameter {} format unsupported".format(
                 name))
-        if not allow_lists and hasattr(values, "__getitem__"):
-            raise NotImplementedError(
-                "Lists of {} are not supported by the implementation of {} on "
-                "this platform".format(name, self.__class__))
 
-    def _check_parameters(self, weights, delays, allow_lists=False):
-        """ Check the types of the weights and delays are supported; lists can\
-            be disallowed if desired.
+    def _check_parameters(self, weights, delays):
```

Comment:
Is this still being used?

Reply:
It seems not (not even in master) - removed